tdvt/tdvt/setup_env.py

```diff
@@ -27,25 +38,113 @@ def create_tdvt_ini_file():
         ini.write('[DEFAULT]\n')
         ini.write('TAB_CLI_EXE_X64 = Full Or Relative/Path/to/tabquerytool.exe\n')
         ini.write('TAB_CLI_EXE_MAC = Full Or Relataive/Path/to/tabquerytool.exe\n')
-        print ("Created ini file: " + ini_path)
-        print ("Please set the tabquery executable file path.")
+        print("Created ini file: " + ini_path)
+        print("Please set the tabquery executable file path.")
     except:
         pass
 
+
+def parse_test_args_to_data_types(
+    custom_table_cols_and_types: List[Dict[str, str]],
+    tableau_data_type_map: Dict[str, str]
+) -> Dict[str, str]:
+    """Parse the column names and data types from the csv file."""
+    col_names = []
+    data_types = []
+    for col in custom_table_cols_and_types:
+        col_names.append(col['name'])
+        data_types.append(col['type'])
+    return dict(zip(col_names, data_types))
+
+
+def parse_custom_schema_csv_file(csv_path):
```

Comment:
Can this be removed?

Reply:
Removed in [09b5058](https://github.com/tableau/connector-plugin-sdk/pull/1101/commits/09b5058187e3153d2c8d3f49f830d91a296293e5)